modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.ts

```diff
@@ -188,13 +188,20 @@ export default class ScenegraphLayer<DataT = any, ExtraPropsT extends {} = {}> e
   };
 
   getShaders() {
-    const modules = [project32, picking, scenegraphUniforms];
+    const defines: {LIGHTING_PBR?: 1} = {};
+    let pbr;
 
     if (this.props._lighting === 'pbr') {
-      modules.push(pbr);
+      pbr = pbrMaterial;
+      defines.LIGHTING_PBR = 1;
```

Comment:
Trying to digest this new define. Seems like more work for the app to have to set this explicitly. But I assume you have a reason for making it explict and don't want to add it to the ShaderModule defines. Perhaps clear in your other PR.

Reply:
Motivated by trying to make the minimal changes to add UBO support. The shaders conditionally used to check for the presence of `MODULE_PBR` to detect if PBR rendering was being used. However, now even in non-pbr mode we need to bind the `pbr_baseColorSampler`, which is coming from the `pbrMaterial` module - which is done with the dummy module defined in deck. 

As the module name is the same, the same define is added in both cases, so I need to change the name of the define. The "more work" is only necessary because the layer is pulling in the PBR module but then also deciding not to use the module when `_lighting` is `flat`